### PR TITLE
implement monitoring API

### PR DESCRIPTION
**Pull request by schiessle**
Implement OCS API to provide some useful system informations

### CODE REVIEW

File: lib/Controller/ApiController.php [hunk 1]
```diff
@@ -20,13 +20,19 @@
  */
 
 
-namespace OCA\ServerInfo;
+namespace OCA\ServerInfo\Controller;
```
Comment (rullzer) [inline lib/Controller/ApiController.php]: `\Controller` was missing :wink:

File: lib/Controller/ApiController.php [hunk 2]
```diff
@@ -20,13 +20,19 @@
...
 
 use OCP\AppFramework\Http\DataResponse;
 use OCP\AppFramework\OCSController;
 
 class ApiController extends OCSController {
 
+
+	public function __construct($appName,
```
Comment (rullzer) [inline lib/Controller/ApiController.php]: It seems the AppFramework needs this.
We should fix it properly. But for now this workaround does the job.

Comment (rullzer) [inline lib/Controller/ApiController.php]: PR to fix this https://github.com/nextcloud/server/pull/509

**Comment by schiessle** [comment]
merged so that @Ivansss and I can work together on the same code basis

**Pull request merged**